### PR TITLE
Feature/end session

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

**Review by angelstackit — APPROVED** [review]
hmm...

File: src/components/EndInterviewSessionButton.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  const dispatch = useAppDispatch()
+  const { interview } = useAppSelector((state) => state.app)
+
+  const endSession = () => {
+    if (interview.status === InterviewStatus.COMPLETED) {
+      return dispatch(openToastInfo('The interview is already completed'))
+    }
+
+    if (interview.status === InterviewStatus.PENDING) {
+      return dispatch(openToastInfo('The interview is not begun yet'))
```
Comment (juliswer) [inline src/components/EndInterviewSessionButton.tsx:17]: Please replace with this text: The interview hasn't started yet.

File: src/pages/components.tsx [hunk 2]
```diff
@@ -1,8 +1,18 @@
 import { BigCalendar } from '@/components/BigCalendar/BigCalendar'
```
Comment (juliswer) [inline src/pages/components.tsx:1]: Delete this file since we are already on dev

**Review by juliswer — APPROVED** [review]
Looks good. Please check my comments before merging!

File: src/pages/components.tsx (hunk 2 above)
Comment (AlefrankM) [reply src/pages/components.tsx:1]: Cool!

File: src/components/EndInterviewSessionButton.tsx (hunk 1 above)
Comment (AlefrankM) [reply src/components/EndInterviewSessionButton.tsx:17]: Perfect

**Pull request merged**